agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/httpclient/LazyHttpClient.java

```diff
@@ -51,7 +51,7 @@
 
 public class LazyHttpClient implements HttpClient {
 
-  private static final String APPLICATIONINSIGHTS_AUTHENTICATION_SCOPE =
+  public static final String APPLICATIONINSIGHTS_AUTHENTICATION_SCOPE =
```

Comment:
I'd suggest just hard-coding this value into the tests instead of making publish. It's not too important tho since this is not part of a public API.